<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>antimindblock</class>
 <widget class="QMainWindow" name="antimindblock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>anti-mindblock</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(49, 49, 49);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="14" column="2">
     <widget class="QLabel" name="enablelabel">
      <property name="text">
       <string>Ctrl+Shift+A = enable</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="scanfolder">
      <property name="text">
       <string>Scan Folder</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="displaylabel">
      <property name="font">
       <font>
        <family>Nunito Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>| display to flip |</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="14" column="3">
     <widget class="QLabel" name="disablelabel">
      <property name="text">
       <string>Ctrl+Shift+D = disable</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="13" column="2">
     <widget class="QPushButton" name="activateaustralia">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(36, 127, 76);</string>
      </property>
      <property name="text">
       <string>ACTIVATE</string>
      </property>
     </widget>
    </item>
    <item row="11" column="3">
     <widget class="QPushButton" name="rotatenumbers">
      <property name="text">
       <string>Rotate Numbers</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="tabletlabel">
      <property name="font">
       <font>
        <family>Nunito Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>| tablet to flip |</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QPushButton" name="rotateScreen">
      <property name="text">
       <string>Rotate Screen</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2" colspan="2">
     <widget class="QComboBox" name="osuskin">
      <property name="styleSheet">
       <string notr="true">border-color: rgb(33, 115, 70);
selection-background-color: rgb(36, 127, 76);</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QComboBox" name="tabletselect">
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(36, 127, 76);</string>
      </property>
     </widget>
    </item>
    <item row="13" column="3">
     <widget class="QPushButton" name="deactivateaustralia">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(174, 64, 37);</string>
      </property>
      <property name="text">
       <string>DEACTIVATE</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="osudirectory"/>
    </item>
    <item row="12" column="3">
     <widget class="QPushButton" name="revertnumbers">
      <property name="text">
       <string>Revert Numbers</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QComboBox" name="displayselect">
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(36, 127, 76);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLabel" name="osudirectorylabel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Nunito Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>osu! directory goes here</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <widget class="QPushButton" name="backupskin">
      <property name="text">
       <string>BACK-UP SKIN</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="osuskinlabel">
      <property name="font">
       <font>
        <family>Nunito Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>osu! skin</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="browse">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="15" column="3">
     <widget class="QPushButton" name="hotkeybutton">
      <property name="text">
       <string>hotkeys</string>
      </property>
     </widget>
    </item>
    <item row="15" column="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Nunito Black</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>enable hotkeys</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>273</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_Config"/>
    <addaction name="actionLoad_Config"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionDisplay">
   <property name="text">
    <string>Display</string>
   </property>
  </action>
  <action name="actionSave_Config">
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
  <action name="actionLoad_Config">
   <property name="text">
    <string>Load Config</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
